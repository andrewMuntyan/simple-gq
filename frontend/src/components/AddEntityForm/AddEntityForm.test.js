// TODO: complete tests

import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import { AddEntityForm } from '.';

describe('<AddEntityForm />', () => {
  const wrapper = shallow(<AddEntityForm />);
  it('renders and displays properly', () => {
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
