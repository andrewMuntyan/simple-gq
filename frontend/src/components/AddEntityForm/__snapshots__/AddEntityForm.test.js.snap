// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddEntityForm /> renders and displays properly 1`] = `
<form
  className="makeStyles-container-1"
  data-test="add-entity-form"
  noValidate={true}
  onSubmit={[Function]}
>
  <span
    className="makeStyles-input-2"
  >
    <TextInput
      disabled={false}
      error={false}
      label={null}
      onChange={[Function]}
      required={true}
      value=""
    />
  </span>
  <WithStyles(ForwardRef(Fab))
    aria-label="add"
    className="makeStyles-fab-3"
    color="secondary"
    disabled={true}
    size="small"
    type="submit"
  >
    <AddIcon />
  </WithStyles(ForwardRef(Fab))>
</form>
`;
